src/ship.js

```
@@ -0,0 +1,21 @@
+
+const Ship = (name, positions) => {
+
+    let hits = [];
+    function hit (position) {
```

As we discussed yesterday, checking for duplicates here would make the Ship more robust.  There are at least 2 ways this could be done: 1) checking explicitly if `position` is already in `hits`, or 2) using a Set 

The `hit` function could either silently do nothing when a duplicate `position` is hit, or it could throw an Exception to alert the caller that they've done something unexpected.  Either way, this Ship module would be more reliable by not being a source of bugs, and not relying on other modules to only send valid input.